pkgs/firehose/bin/health.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:firehose/health.dart';
+
+void main(List<String> arguments) async {
+  var health = Health(Directory.current);
```

Comment:
We could also use `package:args` here - it would provide some validation for us.

```
    ..addMultiOption(
      'checks',
      defaultsTo: ['version', 'license', 'changelog'],
      allowed: ['version', 'license', 'changelog'],
      help: 'Check PR health.',
    )
```